cmd/get_logs.go

```diff
@@ -255,6 +262,13 @@ func (glc *getLogsCmd) collectLogs(node v1.Node, config *ssh.ClientConfig) (stri
 	if err != nil {
 		return stdout, err
 	}
+	if glc.blobServiceSASURL != "" {
+		log.Debug("Will upload logs to storage account")
```

Comment:
I think it is better to upload files after all logs were collected. Otherwise an error uploading files will stop the process.